SUMMARY.md

```diff
@@ -0,0 +1,35 @@
+# Table of contents
+
+* [Project North Star](README.md)
+* [Mechanical](mechanical/README.md)
+  * [Headset BOM](mechanical/headset-bom.md)
+  * [Headset Assembly](mechanical/headset-assembly.md)
+  * [Calibration Stand BOM](mechanical/calibration-stand-bom.md)
+  * [Calibration Stand Assembly](mechanical/calibration-stand-assembly.md)
+  * [Initial Release Assembly Guide](mechanical/initial-release-assembly-guide.md)
+  * [Initial Release 3D Printing Reference](mechanical/initial-release-3d-printing-reference.md)
+  * [Changelog](mechanical/changelog.md)
+* [Software](software.md)
```

Comment:
Ideally the top-level pages are preserved as `README.md`s (so Github's built-in readme system continues to catch it).

Reply:
Done!

Reply:
Arg, sorry, only sort of done. I've tried changing it several times, but gitbook seems to keep reverting it when I make other changes 😕